CHANGELOG.md

```diff
@@ -37,6 +37,7 @@
 - [6819](https://github.com/vegaprotocol/vega/issues/6819) - Support long-living token in wallet client API
 - [6964](https://github.com/vegaprotocol/vega/issues/6964) - Add support for long living tokens with expiry
 - [6991](https://github.com/vegaprotocol/vega/issues/6991) - Expose error field in explorer API
+- [6645](https://github.com/vegaprotocol/vega/issues/6645) - Set decimal number value to be used from oracle instead of from tradable instruments
```

Comment:
I think it's a breaking change isn't it?

Reply:
Yes, sorry I missed that.
Thanks!